### DISCUSSION TITLE
Blocksuite example

### DISCUSSION THREAD

**Discussion by vanillaHill** [toeverything/blocksuite#6207] [Q&A] [answered]
Is there any blocksuite example with component format, like <Editor></Editor> for example?
I found this one but it uses a blocksuite old version.
https://codesandbox.io/p/sandbox/react-blocksuite-starter-nuekpe?file=%2Fsrc%2Fapp.js

**Comment by doouding** [comment, marked as answer]
We just updated our example. I'm not sure if this can meet your needs.
https://github.com/toeverything/blocksuite/tree/master/examples
